test-harness/infratests/integration.go

```diff
@@ -1,5 +1,5 @@
 /*
-This file provides abstractions that simplify the process of integration-testing terraform templates. The goal
+Package infratests This file provides abstractions that simplify the process of integration-testing terraform templates. The goal
```

Comment:
```suggestion
This file provides abstractions that simplify the process of integration-testing terraform templates. The goal
```

Reply:
This is recommended based on the `go-lint` plugin I have installed in VS Code.